mailbox/event/json/src/main/scala/org/apache/james/event/json/EventSerializer.scala

```diff
@@ -279,19 +279,13 @@ class JsonSerialize(mailboxIdFactory: MailboxId.Factory, messageIdFactory: Messa
     case _ => JsError()
   }
   implicit val messageUidReads: Reads[MessageUid] = {
-    case JsNumber(value) => if (value.isValidLong) {
-        JsSuccess(MessageUid.of(value.toLong))
-      } else {
-        JsError()
-      }
+    case JsNumber(value) if value.isValidLong => JsSuccess(MessageUid.of(value.toLong))
+    case JsNumber(_) => JsError()
```

Comment:
you don't need the the case ` case JsNumber(_)` it will be matched in the `case _`

Reply:
True!